markets/perps-market/test/integration/Account/Margins.multiCollateral.test.ts

```diff
@@ -58,7 +58,7 @@ const ethDiscountConfig = {
   upperLimitDiscount: bn(0.05),
   lowerLimitDiscount: bn(0.01),
   discountScalar: bn(2),
-  skewScale: bn(10_000),
+  skewScale: bn(50_000),
```

Comment:
this forces the skew impact to be really small which is where the bug lies otherwise the tests passed cuz the discount would always be higher than the upper limit.